tools/test_history.py

```diff
@@ -297,19 +298,19 @@ def parse_args(raw: List[str]) -> argparse.Namespace:
         help='name of the suite containing the test',
     )
     parser.add_argument(
-        'test',
+        '--test',
         help='name of the test',
     )
     parser.add_argument(
-        'job',
-        nargs='*',
+        '--jobs',
         help='names of jobs to display columns for, in order',
-        default=[],
+        default='',
     )
     args = parser.parse_args(raw)
 
-    args.jobs = None if args.all else args.job
-    if args.jobs == []:  # no jobs, and not None (which would mean all jobs)
+    args.jobs = None if args.all else args.jobs.split(',')
```

Comment:
Instead of making this a comma-separated list, could you use `--action='append'` as described here? https://docs.python.org/3/library/argparse.html#action

Reply:
it's more verbose than I expected (e.g. you need to double type --job many times. but I am ok with this approach as well

Reply:
You're right, it is more verbose; I'd say it's up to you (the `action='append'` is just the one that initially occurred to me)